gef.py

```diff
@@ -837,7 +839,7 @@ def __init__(self, elf: str = "", minimalist: bool = False) -> None:
 
         return
 
-    def read(self, size):
+    def read(self, size: int) -> AnyStr:
```

Comment:
I think is enough since there's only one `open()` call which processes bytes

```suggestion
    def read(self, size: int) -> bytes:
```

Reply:
I always try to include possible future usages as well. But you're right, in most cases this is correct.